src/lib/viewers/controls/color-picker/ColorPickerPalette.tsx

```diff
@@ -1,13 +1,13 @@
 import React from 'react';
+import { AnnotationColor } from '../../../AnnotationModule';
 import './ColorPickerPalette.scss';
 
 export type Props = {
-    onColorSelect: (color: string) => void;
+    colors: Array<AnnotationColor>;
+    onColorSelect: (color: AnnotationColor) => void;
```

Comment:
How about `onSelect`?

Reply:
Yeah that works, color doesn't seem as necessary because this component doesn't have anything else that needs to be selected.